libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -215,6 +215,13 @@ const AP_Param::GroupInfo AP_MotorsMulticopter::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("SAFE_TIME", 42, AP_MotorsMulticopter, _safe_time, AP_MOTORS_SAFE_TIME_DEFAULT),
 
+    // @Param: OPTIONS
+    // @DisplayName: Motor options
+    // @Description: Motor options
+    // @Bitmask: 0:Raw battery voltage compensation
```

Comment:
nitpick: naming the option, "Voltage compensation uses raw voltage" might be better.  As worded it might be misconstrued as turning on/off battery voltage compensation.

Reply:
Fixed thanks